tests/schema.json

```diff
@@ -0,0 +1,45 @@
+{
+    "type": "object",
+    "patternProperties": {
+        "[a-zA-Z0-9]*$": {
+            "type": "object",
+            "properties": {
+                "linux": {
+                    "type": "object",
+                    "properties": {
+                        "root": {
+                            "type": "string"
+                        },
+                        "bfb": {
+                            "type": "string"
+                        },
+                        "symlinks": {
+                            "type": "array",
+                            "items": {
+                                "type": "string"
+                            },
+                            "minItems": 1,
+                            "uniqueItems": true
+                        }
+                    },
+                    "required": [
+                        "root"
+                    ]
+                },
+                "android": {
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    },
+                    "minItems": 1,
+                    "uniqueItems": true
+                }
+            },
+            "additionalProperties": false,
+            "minProperties": 1,
+            "required": [
```

Comment:
@Foggalong The required Linux here will be removed later once we have the Android icons in the data.json file; For now I kept the Linux here to be sure that everything is validated